configs/bundle_config.yaml

```
@@ -141,6 +141,19 @@ databundles:
     - data/hydrobasins/hybas_world_lev04_v1c.shp
     - data/hydrobasins/hybas_world_lev05_v1c.shp
 
+  # global data for hydrobasins
+  bundle_hydrobasins:
+    countries: [Earth]
+    tutorial: false
+    category: common
+    destination: "data/hydrobasins"
+    urls:
+      hydrobasins: https://data.hydrosheds.org/file/HydroBASINS/standard/
+      suffixes: ["af", "ar", "as", "au", "eu", "gr", "na", "sa", "si"]
```

I like the approach!
maybe to be consistent with the others we could do something like:
```suggestion
      hydrobasins:
         url: https://data.hydrosheds.org/file/HydroBASINS/standard/
         suffixes: ["af", "ar", "as", "au", "eu", "gr", "na", "sa", "si"]
```

---

Happy to improve consistency, but not sure I get the point. May you please elaborate a bit? My understanding was that every databundles entry has `urls` as a high-level parameter, which can have additional parameters:

```python
  #   urls:
  #     zenodo: ...
  #     gdrive: ...
```

So, I have tried to follow this pattern: `bundle_name -> urls -> additional parameters`. Do I miss anything?

---

Indeed :)
The purpose of the flag urls would be to give one entry for alternative options to download the same bundle.
That's why I'd be prone to keep one entry (hydrobasins) under urls and potentially add multiple entries underneat, such as:
```suggestion
  bundle_hydrobasins:
    countries: [Earth]
    tutorial: false
    category: common
    destination: "data/hydrobasins"
    urls:
      hydrobasins:
         base_url: https://data.hydrosheds.org/file/HydroBASINS/standard/
         suffixes: ["af", "ar", "as", "au", "eu", "gr", "na", "sa", "si"]
```